controllers/common_controller_utils.go

```diff
@@ -342,13 +342,13 @@ func processDeletedSecrets(ctx context.Context, rc client.Client, req types.Name
 	for _, eachSecret := range allSecrets {
 		if eachSecret.Name == req.Name {
 			// check similarly named secret is present (or not)
-			if secretType := eachSecret.Labels[common.SecretLabelTypeKey]; secretType == string(common.SourceLabel) {
+			if secretType := eachSecret.Labels[utils.SecretLabelTypeKey]; secretType == string(utils.SourceLabel) {
 				// if 'sourceSecretPointer' already points to a source secret,
 				// it is an error. We should not have TWO source
 				// secrets of same name.
 				if sourceSecretPointer != nil {
 					err = errors.New("multiple source secrets detected")
-					logger.Error(err, "Cannot have more than one source secrets with the same name", "request", req, "source-secret", sourceSecretPointer.Name, "namespace", sourceSecretPointer.Namespace)
+					logger.Error(err, "Cannot have more than one source secrets with the same name", "request", req, "source-secret", sourceSecretPointer, "namespace", sourceSecretPointer.Namespace)
```

Comment:
sourceSecretPointer is logging the entire secret object, it leads to secret in logs.

Reply:
changed